<launch>
	<!-- declare arg to be passed in -->
	<arg name="machine" default="localhost" />
	<arg name="user" default="" />

	<machine name="$(arg machine)" address="$(arg machine)" env-loader="/opt/strands/strands_catkin_ws/devel/env.sh" user="$(arg user)" default="true"/>

	<node pkg="topological_patroller" name="patrol_server" type="patrol_server.py" output="screen"/>

</launch>
